web/scripts/build-locales.mjs

```diff
@@ -3,9 +3,16 @@ import fs from "fs";
 import path from "path";
 import process from "process";
 
+/**
+ * Determines if all the Xliff translation source files are present and if the Typescript source
+ * files generated from those sources are up-to-date. If they are not, it runs the locale building
+ * script, intercepting the long spew of "this string is not translated" and replacing it with a
+ * summary of how many strings are missing with respect to the source locale.
+ */
+
 const localizeRules = JSON.parse(fs.readFileSync("./lit-localize.json", "utf-8"));
 
-function compareXlfAndSrc(loc) {
+function translationFileIsUpToDateWithXlf(loc) {
```

Comment:
I didn't intend to update this, but while working on this, one `wireit` pass had a mis-defined lint pass that made this stand out as bad naming.  Both a name change and a comment were warrented.